README.md

```diff
@@ -297,9 +332,9 @@ Add in `app/controllers/application_controller.rb`:
 
 Or if you see better fit, only in the controllers that you will be using unleash.
 
-#### Sample usage
+### 3. Sample usage
 
-Then wherever in your application that you need a feature toggle, you can use:
```

Comment:
There is no requirement to use rails in order to use unleash. The instructions are valid no matter which (if any) framework is used.